docs/book/src/reference/webhook-overview.md

```diff
@@ -0,0 +1,15 @@
+# Webhook
+
+Webhooks are HTTP callbacks, providing a way for notifications to be delivered
+to an external web server. A web application implementing webhooks will send an
```

Comment:
This explanation confuses things.  They're not really "notifications" (which imply fire-and-forget on the part of the caller) -- they're requests for information sent in a blocking fashion.